src/main/cpp/t07_quad2.cpp

```diff
@@ -21,5 +21,34 @@
 using namespace std;
 
 int t07_quad2() {
-
+    double a, b, c, x1, x2, D;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'x2' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4353647)